polaris-icons/package.json

```diff
@@ -63,5 +63,8 @@
     "js-yaml": "^4.0.0",
     "rehype-parse": "^7.0.1",
     "unified": "^9.2.1"
+  },
+  "peerDependencies": {
+    "react": "*"
   }
```

Comment:
Some consumers of `@shopify/polaris-icons` might not use React and instead use the raw SVG assets. Should we make this an [optional peer dependency](https://pnpm.io/next/package_json#peerdependenciesmetaoptional)?

```suggestion
  },
  "peerDependenciesMeta": {
    "react": {
      "optional": true
    }
  }
```

Reply:
Good suggestion @jesstelford and thanks for updating this @ryanwilsonperkin. That strategy sounds good to me